protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -111,80 +111,81 @@ Select {
+5. Create an oracle that calls a read method of a smart contract and specify an incorrect ABI format for the event. Proposal should fail validation and should return an error (<a name="0082-ETHD-023" href="#0082-ETHD-023">0082-ETHD-023</a>)
 6. Will need some tests around consensus, will require setting up a network and having some nodes receive different values for the same oracle data point and testing that the oracle data point is/is not published depending on voting (<a name="0082-ETHD-024" href="#0082-ETHD-024">0082-ETHD-024</a>)
+7. Create 2 markets with exactly the same ethereum oracle data sources for termination and settlement. Verify that the data spec ID are the same and the data sources are ACTIVE. Now terminate and settlement Market1 using governance proposals. Now the status of the data sources should still be ACTIVE as MArket2 is still using them.
+8. Ensure that when a market is suspended and then resumed via a governance proposal we can still terminate and settle the market using ethereum oracle.
 
 ### API
 
-1. Ability to query oracle data sources via an API endpoint - filters should be available for data source - internal OR external, status - Active / Inactive / Expired (<a name="0082-ETHD-025" href="#0082-ETHD-025">0082-ETHD-025</a>)
-2. Ability to query historic data sent by an oracle data source (<a name="0082-ETHD-026" href="#0082-ETHD-026">0082-ETHD-026</a>)
+1. Ability to query oracle data sources via an API endpoint (REST, gRPC and graphQL) - filters should be available for data source - internal OR external, status - Active / Inactive / Expired (<a name="0082-ETHD-025" href="#0082-ETHD-025">0082-ETHD-025</a>)
+2. Ability to query historic data sent by an oracle data source and processed by vega network (<a name="0082-ETHD-026" href="#0082-ETHD-026">0082-ETHD-026</a>)
 
 ### Non Functional
 
 1. Network wide contract error should be reported via oracle data events (<a name="0082-ETHD-027" href="#0082-ETHD-027">0082-ETHD-027</a>)
 2. Phase 2 - System needs to emit an error via the TX RESULT event if the data source does NOT emit events in a timely fashion. e.g. if the data source is expected to emit events every 5 minutes and if we do not receive 3 consecutive events , then raise an error via the TX RESULT event (<a name="0082-ETHD-028" href="#0082-ETHD-028">0082-ETHD-028</a>)
 3. Phase 2 - Define behaviour for missed events / missed scheduled smart contract calls - e.g. if an oracle data source is scheduled to emit events every 10 minutes and we miss 5 events because of protocol upgrade or some other outage - then do we catch up those events or skip those events ? Maybe this is defined in the oracle data source definition (<a name="0082-ETHD-029" href="#0082-ETHD-029">0082-ETHD-029</a>)
```

Comment:
same as this [comment](https://github.com/vegaprotocol/specs/pull/1875/files#r1284248380)

```suggestion
3. Phase 2 - Define behaviour for missed events / missed scheduled smart contract calls - e.g. if an oracle data source is scheduled to emit events every 10 minutes and we miss 5 events because of protocol upgrade or some other outage - then do we catch up those events or skip those events ? Maybe this is defined in the oracle data source definition (<a name="0082-PLAZZO-029" href="#0082-PLAZZO-029">0082-PLAZZO-029</a>)
```

Reply:
AC numbering updated.